test/conformance/queue/urQueueCreate.cpp

```diff
@@ -78,13 +78,17 @@ TEST_P(urQueueCreateTest, InvalidValueProperties) {
 
 TEST_P(urQueueCreateTest, InvalidQueueProperties) {
     ur_queue_handle_t queue = nullptr;
-
-    // It should be an error to specify both low/high priorities
     ur_queue_properties_t props = {
         /*.stype =*/UR_STRUCTURE_TYPE_QUEUE_PROPERTIES,
         /*.pNext =*/nullptr,
-        /*.flags =*/UR_QUEUE_FLAG_PRIORITY_HIGH | UR_QUEUE_FLAG_PRIORITY_LOW,
+        /*.flags =*/UR_QUEUE_FLAG_PRIORITY_HIGH | UR_QUEUE_FLAG_PRIORITY_LOW |
+            UR_QUEUE_FLAG_SUBMISSION_BATCHED |
+            UR_QUEUE_FLAG_SUBMISSION_IMMEDIATE,
     };
+    // It should be an error to specify both low/high priorities
+    ASSERT_EQ_RESULT(UR_RESULT_ERROR_INVALID_QUEUE_PROPERTIES,
+                     urQueueCreate(context, device, &props, &queue));
+    // It should be an error to specify both batched and immediate submission
     ASSERT_EQ_RESULT(UR_RESULT_ERROR_INVALID_QUEUE_PROPERTIES,
                      urQueueCreate(context, device, &props, &queue));
```

Comment:
This is asserting the same thing twice, instead the `prop.flags` in each call to `urQueueCreate` should be testing the flags the comments actually talk about:

```suggestion
    // It should be an error to specify both low/high priorities
    ur_queue_properties_t props = {
        /*.stype =*/UR_STRUCTURE_TYPE_QUEUE_PROPERTIES,
        /*.pNext =*/nullptr,
        /*.flags =*/UR_QUEUE_FLAG_PRIORITY_HIGH | UR_QUEUE_FLAG_PRIORITY_LOW,
    };
    ASSERT_EQ_RESULT(UR_RESULT_ERROR_INVALID_QUEUE_PROPERTIES,
                     urQueueCreate(context, device, &props, &queue));
    // It should be an error to specify both batched and immediate submission
    props.flags =
        UR_QUEUE_FLAG_SUBMISSION_BATCHED | UR_QUEUE_FLAG_SUBMISSION_IMMEDIATE;
    ASSERT_EQ_RESULT(UR_RESULT_ERROR_INVALID_QUEUE_PROPERTIES,
                     urQueueCreate(context, device, &props, &queue));
```

Reply:
I changed the test to separately test low/high and batched/immediate slightly differently than the suggestion to avoid clobbering an existing queue object.

Reply:
Thanks, if you could also run `clang-format` on this file that should fix the failed GitHub Actions.

Reply:
Formatting change has been made.